ansible/roles/csm.sbps.apply_label/files/apply_k8s_label.sh

```diff
@@ -26,4 +26,7 @@
 LABEL="iscsi=sbps"
 HOST_NAME="$(awk '{print $1}' /etc/hostname)"
 
-kubectl label nodes $HOST_NAME $LABEL
+if ! kubectl get nodes -l $LABEL | grep -q "$HOST_NAME"
+then
+  kubectl label nodes "$HOST_NAME" $LABEL
+fi
```

Comment:
```suggestion
# If the label is already applied to this node, then exit
kubectl get nodes -l "$LABEL" --no-headers | grep -Eq "^${HOST_NAME}[[:space:]]" && exit 0

# Otherwise, apply the label to the node
kubectl label nodes "$HOST_NAME" "$LABEL"
```